[v3] CodeSandbox is missing @types/node dependency

```bash
[ERROR] 07:59:17 ⨯ Unable to compile TypeScript:
src/main.ts(2,30): error TS2307: Cannot find module 'node:http' or its corresponding type declarations.
```

--- a/examples/issue-template/package.json
+++ b/examples/issue-template/package.json
@@ -3,13 +3,14 @@
   "version": "1.13.11",
   "private": true,
   "devDependencies": {
-    "typescript": "4.8.4",
+    "@types/node": "18.11.9",
+    "ts-node": "10.9.1",
     "ts-node-dev": "2.0.0",
-    "ts-node": "10.9.1"
+    "typescript": "4.8.4"
   },
   "dependencies": {
-    "graphql-yoga": "^3.0.0-next.10",
-    "graphql": "^16.3.0"
+    "graphql": "^16.3.0",
+    "graphql-yoga": "^3.0.0-next.10"
   },
   "scripts": {
     "dev": "ts-node-dev src/main.ts",

